src/main/webapp/WEB-INF/web.xml

```diff
@@ -27,6 +27,11 @@
         <lookup-name>java:jboss/datasources/FlowableDS</lookup-name>
     </resource-ref>
 
+    <filter-mapping>
```

Comment:
Zet deze even wat dichter op de UserPrincipalFilter mapping en zet in commentaar dat de volgorde hier relevant is